src/main/java/seedu/duke/food/FoodList.java

```diff
@@ -0,0 +1,89 @@
+package seedu.duke.food;
+
+import java.util.ArrayList;
+
+public class FoodList {
+    public static final String LS = System.lineSeparator();
+    public static final String MESSAGE_FOOD_LEFT_COUNT = "There %s %d food left" + LS;
+    public static final String MESSAGE_FOOD_COUNT = "You have consumed %d food items:" + LS;
+    public static final String MESSAGE_FOOD_CLEAR = "All food items have been deleted.";
+    public static final String MESSAGE_EMPTY_FOOD_LIST = "\tThere is no record in the food list.";
+    public static final String MESSAGE_FOOD_DELETION = "You have removed this food item: " + LS + "\t%s" + LS;
```

Comment:
hello ruiyang, let's try and standardize our static final messages to be printed so that we are able to make the app look more unified and coherent